osu.Game.Rulesets.Mania/UI/DrawableManiaRuleset.cs

```diff
@@ -55,6 +55,11 @@ public class DrawableManiaRuleset : DrawableScrollingRuleset<ManiaHitObject>
         // Stores the current speed adjustment active in gameplay.
         private readonly Track speedAdjustmentTrack = new TrackVirtual(0);
 
+        /// <summary>
+        /// Whether difficulty points should be taken into account, even on convert maps.
+        /// </summary>
+        public bool PreserveDifficultyPoints { get; set; }
```

Comment:
Re-thinking this one, i really don't think this should exist..

Reply:
I agree and was reluctant to suggest it in the first place, but in the grand scheme of things I don't deem this pull important enough to spend time on doing things properly by substituting the scroll algorithm.